packages/fxa-auth-server/package.json

```diff
@@ -179,7 +179,6 @@
     "esbuild": "^0.14.2",
     "esbuild-register": "^3.2.0",
     "eslint": "^7.32.0",
-    "eslint-plugin-fxa": "^2.0.2",
     "fxa-shared": "workspace:*",
```

Comment:
Unrelated and barely interesting, but this is duplicated on L96 above as a dependency as well:

https://github.com/mozilla/fxa/blob/3e749085c9207a7a1724a3dcb15c0a774838fe14/packages/fxa-auth-server/package.json#L96

(I was wondering why my `git grep "workspace:" packages/*/package.json | sort | uniq | wc -l` returned off-by-one results depending on whether I used `uniq` or not.)

Reply:
Just a side note, I think in general we could revisit our dev vs non-dev dependencies... not sure if there is a way to automate this.